examples/mcu-board-support/stm32h735g.rs

```diff
@@ -285,19 +280,45 @@ impl slint::platform::Platform for StmBackend {
         // Init Timer
         let mut timer = dp.TIM2.tick_timer(10000.Hz(), ccdr.peripheral.TIM2, &ccdr.clocks);
         timer.listen(hal::timer::Event::TimeOut);
-        self.timer.set(timer).unwrap();
 
         // Init Touch screen
         let scl =
             gpiof.pf14.into_alternate::<4>().set_open_drain().speed(High).internal_pull_up(true);
         let sda =
             gpiof.pf15.into_alternate::<4>().set_open_drain().speed(High).internal_pull_up(true);
-        let mut touch_i2c =
-            dp.I2C4.i2c((scl, sda), 100u32.kHz(), ccdr.peripheral.I2C4, &ccdr.clocks);
+        let touch_i2c = dp.I2C4.i2c((scl, sda), 100u32.kHz(), ccdr.peripheral.I2C4, &ccdr.clocks);
+
+        Self {
+            window: RefCell::default(),
+            scb: RefCell::new(cp.SCB),
+            delay: RefCell::new(delay),
+            timer,
+            layer: RefCell::new(layer),
+            touch_i2c: RefCell::new(touch_i2c),
+        }
+    }
+}
 
-        let mut ft5336 = ft5336::Ft5336::new(&mut touch_i2c, 0x70 >> 1, &mut delay).unwrap();
+impl slint::platform::Platform for StmBackend {
+    fn create_window_adapter(
+        &self,
+    ) -> Result<Rc<dyn slint::platform::WindowAdapter>, slint::PlatformError> {
+        let window = slint::platform::software_renderer::MinimalSoftwareWindow::new(
+            slint::platform::software_renderer::RepaintBufferType::SwappedBuffers,
+        );
+        self.window.replace(Some(window.clone()));
+        Ok(window)
+    }
+
+    fn run_event_loop(&self) -> Result<(), slint::PlatformError> {
+        let mut touch_i2c = self.touch_i2c.borrow_mut();
+
+        let mut delay = self.delay.borrow_mut();
+        let mut ft5336 = ft5336::Ft5336::new(&mut *touch_i2c, 0x70 >> 1, &mut *delay).unwrap();
         ft5336.init(&mut touch_i2c);
 
+        let (fb1, fb2) = unsafe { (&mut FB1, &mut FB2) };
```

Comment:
Is that sound?
What does prevent run_event_loop to be called several time recursively?
Ah, the borrow_mut of the delay and touch_i2c that can exist only once.   Maybe this deserve a "Safety" comment.

I wonder if it wouldn't be cleaner to put them in a RefCell in `StmBackend`